rero_ils/modules/notifications/dispatcher.py

```diff
@@ -236,7 +236,7 @@ def send_mail_for_printing(notifications=None):
         # 3. Send the message
         msg = Dispatcher._create_email(
             recipients=[recipient],
-            reply_to=reply_to,
+            reply_to=[reply_to],
```

Comment:
Here we will put the string into a list and get:
`['person1@gmail.com, person2@gmail.com']`
A list with one string. Should we not use ` isinstance(reply_to, list)` or always put a list into `reply_to?

Reply:
in this method, I was sure than `reply_to = library.get('email')` return a string (Check library JSON schema). To work same between "send_by_mail" and "send_for_printing", I force "reply_to" to be a list containing only one address.